lms/djangoapps/grades/signals/handlers.py

```diff
@@ -100,6 +104,7 @@ def submissions_score_reset_handler(sender, **kwargs):  # pylint: disable=unused
         course_id=course_id,
         usage_id=usage_id,
         modified=kwargs['created_at'],
+        score_deleted=True,
```

Comment:
This parameter should have been present from the beginning - a `score_reset` will always result in a "hidden" 0/0 score existing and triggering the "hidden score is None" experience in our task.